Vertex palette install update (#1464)
spectrocloud/librarium: content/docs/12.5-vertex/02-install-palette-vertex/10-install-on-vmware/10-install.md

# Install the Enterprise Cluster

> The video below provides a demonstration of the installation wizard and the prompts you will encounter. Take a moment to watch the video before you begin the installation process. Make sure to use values that are appropriate for your environment. Use the three-dots icon in the lower right corner of the video to expand the video to full screen and to change the playback speed.

Author: ```suggestion
The video below provides a demonstration of the installation wizard and the prompts you will encounter. Take a moment to watch the video before you begin the installation process. Make sure to use values that are appropriate for your environment. Use the **three-dot Menu** in the lower right corner of the video to expand the video to full screen and to change the playback speed.
```

  | **Registry Password** or **Registry Secret Key** | Enter the registry password or the secret key if using `OCI ECR`. |
  | **Registry Region** | Enter the registry region. This option is only available if you are using `OCI ECR`. |
  | **ECR Registry Private** | Type `y` if the registry is private. Otherwise, type `n`. |
  | **Use Registry for Images** | Type `y` to use a different registry for images. Type `n` to use the same registry for images. If you are using another registry for images, you will be prompted to enter the registry URL, base path, username, and password. |

<br />

10. Next, specify the database storage size to allocate for Palette VerteX. The default is 20 GB. Refer to the [size guidelines](/vertex/install-palette-vertex#sizeguidelines) for additional information.

11. The next set of prompts is for the VMware vSphere account information. Enter the information listed in the following table.

  <br />

  #### VMware vSphere Account Information

  |**Parameter**                            | **Description**|
  |-----------------------------------------|----------------|

>   |**vSphere Endpoint** | vSphere endpoint. Must be a fully qualified domain name (FQDN) or IP address without a scheme - that is, without an IP protocol, such as `https://`. Example: `vcenter.mycompany.com`.|
>   |**vSphere Username** | vSphere account username.|
>   |**vSphere Password**| vSphere account password.|

Author: ```suggestion
  |**vSphere Endpoint** | VMware vSphere endpoint. Must be a fully qualified domain name (FQDN) or IP address without a scheme - that is, without an IP protocol, such as `https://`. Example: `vcenter.mycompany.com`.|
  |**vSphere Username** | VMware vSphere account username.|
  |**vSphere Password**| VMware vSphere account password.|
```

>   |**Allow Insecure Connection** | Bypasses x509 verification. Type `Y` if using a vSphere instance with self-signed Transport Layer Security (TLS) certificates. Otherwise, type `n`.|

Author: ```suggestion
  |**Allow Insecure Connection** | Bypasses x509 verification. Type `Y` if using a VMware vSphere instance with self-signed Transport Layer Security (TLS) certificates. Otherwise, type `n`.|
```
